make docs-serve error on Red Hat 7

**Describe the bug**
make docs-serve fails on Red Hat 7.

**To Reproduce**
Steps to reproduce the behavior:
1. Find Red Hat 7 image
2. Clone compliance-trestle repo
3. Run: make develop
4. Run: git submodule update --init
5. Run: make docs-serve

**Expected behavior**
trestle docs webserver to run.

**Screenshots**
(venv.compliance-trestle) [degenaro@myhost compliance-trestle]$ make docs-serve
python ./scripts/website_automation.py
Traceback (most recent call last):
  File "./scripts/website_automation.py", line 87, in <module>
    md_txt(pathlib.Path('license'), pathlib.Path('docs/license.md'))
  File "./scripts/website_automation.py", line 74, in md_txt
    assert original.exists()
AssertionError
make: *** [docs-automation] Error 1

**Desktop (please complete the following information):**
 - OS: Red Hat Enterprise Linux Workstation 7.9 (64-bit)

--- a/scripts/website_automation.py
+++ b/scripts/website_automation.py
@@ -84,5 +84,5 @@ if __name__ == '__main__':
     structer = create_module_markdowns(pathlib.Path('trestle'), 'trestle', pathlib.Path('docs/api_reference'))
     update_mkdocs_meta(pathlib.Path('mkdocs.yml'), structer)
     # Ensure single source of truth for license file
-    md_txt(pathlib.Path('license'), pathlib.Path('docs/license.md'))
+    md_txt(pathlib.Path('LICENSE'), pathlib.Path('docs/license.md'))
     md_txt(pathlib.Path('DCO1.1.txt'), pathlib.Path('docs/contributing/DCO.md'))

